harvester/edurep/models.py

```diff
@@ -51,8 +51,8 @@ def __str__(self):
 
 class EdurepHarvest(models.Model):
 
-    source = models.ForeignKey(EdurepSource)
-    freeze = models.ForeignKey(Freeze)
+    source = models.ForeignKey(EdurepSource, on_delete=models.SET_NULL, null=True, blank=True)
+    freeze = models.ForeignKey(Freeze, on_delete=models.SET_NULL, null=True, blank=True)
```

Comment:
This is a through model and it wouldn't make sense to keep the model around if the freeze or source gets deleted. So models.CASCADE (as is the default before 2.2)